src/device-registry/models/KnowYourAirUserQuizProgress.js

```
@@ -115,11 +115,13 @@
       const inclusionProjection =
         constants.KYA_QUIZ_PROGRESS_INCLUSION_PROJECTION;
       const exclusionProjection = constants.KYA_QUIZ_PROGRESS_EXCLUSION_PROJECTION(
-        filter.category ? filter.category : "none"
+        filter.path ? filter.path : "none"
       );
-      if (!isEmpty(filter.category)) {
-        delete filter.category;
+
+      if (!isEmpty(filter.path)) {
+        delete filter.path;
       }
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)